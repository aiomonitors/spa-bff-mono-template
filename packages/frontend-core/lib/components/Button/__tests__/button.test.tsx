import React from 'react';
import { renderToString } from 'react-dom/server';
import { Button } from '..';

describe('Button Component', () => {
  it('Should match the snapshot', () => {
   const item = renderToString(<Button />);

   expect(item).toMatchSnapshot();
  })
})
