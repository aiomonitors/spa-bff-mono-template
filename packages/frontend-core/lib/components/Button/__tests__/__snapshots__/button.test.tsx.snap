// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Component Should match the snapshot 1`] = `"<button type="button">This is a button</button>"`;
